python/lib/sift_py/ingestion/rule/config.py

```diff
@@ -26,19 +26,21 @@ class RuleConfig(AsJson):
     name: str
     description: str
     expression: str
-    action: RuleAction
+    action: Optional[RuleAction]
     channel_references: List[ExpressionChannelReference]
 
     def __init__(
         self,
         name: str,
-        description: str,
-        expression: str,
-        action: RuleAction,
         channel_references: List[
             Union[ExpressionChannelReference, ExpressionChannelReferenceChannelConfig]
         ],
+        description: str = "",
+        expression: str = "",
+        action: Optional[RuleAction] = None,
```

Comment:
Is there a better default value for this than `None`? Otherwise this is the only one that must be explicitly `Optional`.